SimplifiedTests/Mocks/NYPLAnnotationsMock.swift

```diff
@@ -0,0 +1,160 @@
+//
+//  NYPLAnnotationsMock.swift
+//  SimplyETests
+//
+//  Created by Ernest Fan on 2021-12-07.
+//  Copyright © 2021 NYPL Labs. All rights reserved.
+//
+
+import Foundation
+import R2Shared
+@testable import SimplyE
+
+class NYPLAnnotationsMock: NYPLAnnotationSyncing {
+  static var failRequest: Bool = false
+  static var serverBookmarks: [String: [NYPLReadiumBookmark]] = [String: [NYPLReadiumBookmark]]()
+  static var readingPositions: [String: NYPLBookmarkSpec] = [String: NYPLBookmarkSpec]()
+  
+  // For generating unique annotation id
+  static private var annotationCounter: Int = 0
+  
+  // Server status
+  
+  static func requestServerSyncStatus(forAccount userAccount: NYPLUserAccount,
+                                      completion: @escaping (_ enableSync: Bool) -> ()) {
+    completion(true)
+  }
+  
+  static func updateServerSyncSetting(toEnabled enabled: Bool, completion:@escaping (Bool)->()) {
+    completion(false)
```

Comment:
here we are setting up the mock to always mock a failure, correct?  just checking if this is the intention.

Reply:
This is actually some place holder code when I first implemented this mock class, this and a few other functions are not needed for the unit tests, but they are required due to the `NYPLAnnotationSyncing` protocol. I can remove the body of the functions if that's more appropriate.

Reply:
got it, and no it’s fine as is.